index.html

```diff
@@ -0,0 +1,249 @@
+                        <a><i class="fa-brands fa-twitter"></i></a>
+                        <a>English</a>
+                        <a>Sign Up</a>
+                        <a>Login</a>
+                    </div>
+                </header>
+                <nav class="">
+                    <div class="desktop-menu">
+                        <div class="header-logo"><img src="JM-Images/JunglemanicLogo.png"></div>
+                        <div><a class="about-h4" href="/pages/about.html">About</a></div>
+                        <div>
+                            <h4><a href="#activities">Main Activities</a></h4>
+                        </div>
+                        <div>
+                            <h4><a href="#single-activity-section">JM Activity</a></h4>
+                        </div>
+                        <div>
+                            <h4><a href="#contact-form">Contact Us</a></h4>
+                        </div>
+                        <div>
+                            <h4 class="customized-navitem">thejunglemaniac</h4>
+                        </div>
+                    </div>
+                    <div class="hamburger-icon">
+                        <i class="fas fa-bars fa-lg"></i>
+                    </div>
+                </nav>
+                <div class="mobile-menu-list">
+                    <div class="x-closeicon-wrapper">
+                        <img class="x-close-icon" src="JM-Images/Icons/Cancel-Icon.png" alt="Close-Icon-Image">
+                    </div>
+                    <ul>
+                        <li> <a href="#activities">Main Activities</a></li>
+                        <li> <a href="/pages/about.html">About</a></li>
+                        <li> <a href="#single-activity-section">JM Activity</a></li>
+                        <li> <a href="#contact-form">Contact Us</a></li>
+                    </ul>
+                </div>
+            </div>
+
+            <section class="top-section">
+                <div class="topsection-overlay">
+                    <div class="top-section-container">
+                        <div class="section-one-content">
+                            <h4>thejunglemaniac</h4>
+                            <h1>Explore with us ...</h1>
+                        </div>
+
+                        <div class="section-two-container">
+                            <p>Join our community page as we explore Pokhara and its surroundings through hiking,
+                                camping, cycling, and motorcycle rides. We value nature, preserving it through
+                                sustainable
+                                practices. Experience thrilling adventures, connect with like-minded individuals, and
+                                create
+                                lasting memories in this vibrant and nature-centered community.
+                            </p>
+                        </div>
```

Comment:
Currently your website is missing a lot of the styles from the behance design guidelines. Please go through all the points listed below and update your website to conform to the behance design:

Your design:
<img width="434" alt="image" src="https://github.com/pushkar1995/capstone-project1.0/assets/37116322/74642cea-11db-4edd-8486-1ba0b56498bc">

Behance design:
<img width="453" alt="image" src="https://github.com/pushkar1995/capstone-project1.0/assets/37116322/d17cfd3b-4fb5-4dcc-8112-2fb3d6194977">


- Kindly update the main color of your website to be the same color used in the behance design. The main color used is `#fe5038`
- Currently there is a horizontal scroll in the desktop version of your website. Please identify the element causing it and remove it
- Kindly increase the font size and weight of the main title in the headline section  to match the behance design
-Kindly add some padding to each card in the mobile version of the main program section and align the text of each text to be in line with the behance design
- Kindly remove the activities and contact section. There are not present in the behance design
- Kindly make sure that each picture of your featured speakers are the same size
- Kindly add the partner section present in the behance design
- Kindly update the footer in your mobile version to have the logo aligned to the left and the text aligned to the right